packages/orbit-components/src/Wizard/index.js.flow

```diff
@@ -0,0 +1,19 @@
+// @flow
+import * as React from "react";
+
+import type { Globals } from "../common/common.js.flow";
+import typeof WizardStepComponent from "./WizardStep";
+
+export type Status = "available" | "completed" | "disabled";
```

Comment:
Consider to move to Context, because you use it only there and in icon component :)